chord_metadata_service/chord/workflows/wdls/document.wdl

```diff
@@ -30,15 +27,16 @@ task post_to_drs {
     input {
         File file_path
         String drs_url
-        String project_id
-        String dataset_id
+        String project_dataset
         String token
     }
     command {
+        project_id=$(python3 -c 'print("~{project_dataset}".split(":")[0])')
+        dataset_id=$(python3 -c 'print("~{project_dataset}".split(":")[1])')
         curl -k -X POST \
```

Comment:
I think we should also inject a tag of some sort that indicates if the workflow requests should be secure (PROD) or insecure because of self-signed certs (DEV). Right now all curl commands use the `-k` flag no matter what.

Reply:
Here we could also validate the http code to avoid false workflow completions with 4XX codes